<launch>
    <arg name="planning_scene_config" default="/root/ocla/catkin_ws/src/kek_routines/config/UR_grinding_demo/planning_scene_config_sound_sensor.yaml"/>
    <arg name="motion_config" default="/root/ocla/catkin_ws/src/kek_routines/config/UR_grinding_demo/motion_config.yaml"/>
    <arg name="abs_force" default="true"/> 
    <arg name="rosbag_record" default="false"/>
    <arg name="rosbag_name" default=""/>

    <node name="grinding_demo" pkg="kek_routines" type="grinding_demo.py" output="screen">    
        <param name="move_group_name" value="manipulator" />
        <param name="grinding_eef_link" value="pestle_tip" />
        <param name="gathering_eef_link" value="spatula_tip" />
        <rosparam command="load" file="$(arg planning_scene_config)"/>
        <rosparam command="load" file="$(arg motion_config)"/>
    </node>

    <!-- <group if="$(arg abs_force)">
        <node name="abs_force_pub" pkg="kek_routines" type="abs_force_publisher.py" output="screen" />
    </group> -->

    <group if="$(arg rosbag_record)">
        <node pkg="rosbag" type="record" name="rosbag_record"
        args='record -a -o $(arg rosbag_name) -x "/camera/depth/(.*)"'/>
    </group>
</launch>